interface/changelog.dm

```diff
@@ -0,0 +1,19 @@
+/datum/changelog
```

Comment:
Move this somewhere in `src` please

Reply:
Could you please elaborate further how exactly you want the change to be like?

Reply:
Okay, I believe I figured it out. If not, please re-review